app/models/katello/content_view.rb

```diff
@@ -76,6 +74,7 @@ def <<(*_args)
 
     scoped_search :on => :name, :complete_value => true
     scoped_search :on => :organization_id, :complete_value => true
+    scoped_search :on => :composite, :complete_value => true
```

Comment:
Composite does not seem to auto-complete properly to show 'false'.  It only autocompletes if there is a 'true' value that exists.  You may want to explicitly set the auto-complete values for that field, which i think you can do.